### DISCUSSION TITLE
Initial population as input

### DISCUSSION THREAD

**Discussion by ignaspakamore** [PasaOpasen/geneticalgorithm2-6.8.7#14] [Show and tell]
Hi, 

Thanks for this amazing code!

I was wondering would it be possible to use initial population created outside _geneticalgorithm2()_ and pass it via run function?

**Comment by PasaOpasen** [comment]
@ignaspakamore sure:

https://github.com/PasaOpasen/geneticalgorithm2#how-to-initialize-start-population-how-to-continue-optimization-with-new-run
